app/src/main/java/co/nimblehq/compose/crypto/ui/screens/home/HomeViewModel.kt

```diff
@@ -103,6 +111,42 @@ class HomeViewModel @Inject constructor(
         _showTrendingCoinsLoading.value = false
     }
 
+    @OptIn(FlowPreview::class)
```

Comment:
redundant on my end 😛 

![image](https://user-images.githubusercontent.com/16315358/198540429-c9911a1d-90da-4a39-98a4-70b864c3b0c7.png)

Reply:
Hmmm... AS asked me to add this before. Let's me remove it.